CMSgov/cms-carts-seds
# MDCT-370: Header Update

# Description

Fixes [MDCT-370](https://qmacbis.atlassian.net/browse/MDCT-370). Reformats the header to look super snazzy!

Screen shots:
![Screen Shot 2022-06-09 at 4 47 01 PM](https://user-images.githubusercontent.com/19439679/172941635-6d98d0c3-e173-4182-8860-f2567069e82b.png)

![Screen Shot 2022-06-09 at 4 47 32 PM](https://user-images.githubusercontent.com/19439679/172941644-050e74e4-f4ee-4474-84df-e9c6e0a63f9c.png)

![Screen Shot 2022-06-13 at 10 27 23 AM](https://user-images.githubusercontent.com/19439679/173376459-b66f0360-4373-4278-8dfc-c49c39f8850d.png)


## How to test

Load up Carts and play around with the header. It should

1. Stick to the top of the page
2. Only show the autosave banner for State Users on an In-Progress/Uncertified/New report.
3. Have new Icon and swap to My Account for dropdown nav
4. Have new Icon and Get Help section
5. Have a flashy, elegant, regal new type treatment in the top left 
6. Better Mobile Design

## Dependencies

Please spell out any dependencies for this change -- ex -- requires an install / update / migration etc

# Get it done

Now that the PR is open this is what happens next

## Code authors checklist


## Reviewers Checklist (Two different people)


## Assignee
